README.md

```diff
@@ -85,18 +85,27 @@ each of your installed apps.  For example, if you have an app named
 `customers.tasks`:
 
 ``` python
-import dramatiq
-
 from django.core.mail import send_mail
+from django_dramatiq.utils import actor
 
 from .models import Customer
 
-@dramatiq.actor
+@actor
```

Comment:
**issue:** I dont think we should be introducing a new `actor` method here different from the core library. I could see this becoming confusing as the APIs/Documentation would diverge. Is there any way to introduce this to the core library?